lib/src/base-styles/_nx-lists.scss

```diff
@@ -142,6 +146,11 @@
       background-color: #d3e6ff;
     }
 
+    &:focus {
+      box-shadow: 0 0 3px 1px rgba(2, 166, 217, 0.3);
+      border: solid 1px #02a6d9;
```

Comment:
I think this could just be `border-color: #02a6d9`

Reply:
> I think this could just be `border-color: #02a6d9`

https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/e57ab0da139398c2561b16f90a4e9976f7da0c73